debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/connection/ReplicationConnectionIT.java

```diff
@@ -61,7 +62,7 @@ public void shouldCreateAndDropReplicationSlots() throws Exception {
         }
     }
 
```

Comment:
I think you need to add test that the connection is closed if the replication slot name is invalid (for example: it contains `-` character)